### PR TITLE
Nginx reverse proxy setting

### PR DESCRIPTION
**Pull request by bigbugcc**
Reverse proxy root path or subpath web panel

### CODE REVIEW

File: README.ru_RU.md [hunk 1]
```diff
@@ -201,6 +201,41 @@ systemctl restart x-ui
...
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header Range $http_range;
+    proxy_set_header If-Range $http_if_range; 
+    proxy_redirect off;
+    proxy_pass http://127.0.0.1:2053;
+}
+```
+
+#### Nginx sub-path
+- Убедитесь, что «корневой путь url панели» в настройках панели `/sub` совпадает.
```
Comment (voronin9032) [inline README.ru_RU.md]: > «корневой путь url панели»

Actually that's named "корневой путь URL адреса панели" in the Russian translation of the Panel. It's better to call it as it's called in the Panel -- "корневой путь URL адреса панели".

Comment (voronin9032) [reply README.ru_RU.md]: Also, that line sounds a bit odd. A better translation would be: ```- Убедитесь, что "корневой путь URL адреса панели" в настройках панели и `/sub` совпадают.```

Comment (bigbugcc) [reply README.ru_RU.md]: O, thanks for the heads up, I fixed the translation here.

**Comment by MHSanaei** [comment]
thank you

**Pull request merged**